python/tests/test_exporter.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+import numpy as np
+from unittest.mock import patch, Mock
```

Comment:
`patch` and `Mock` should be imported from the mock library (the python2.7 backport)